crates/apollo-mcp-server/src/apps.rs

```diff
@@ -218,6 +225,50 @@ pub(crate) fn load_from_path(
     Ok(apps)
 }
 
+fn merge_inputs(orig: &mut Map<String, Value>, extra: Vec<ExtraInputDefinition>) {
+    let mut properties = orig
+        .remove("properties")
+        .and_then(|val| {
+            if let Value::Object(properties) = val {
+                Some(properties)
+            } else {
+                None
+            }
+        })
+        .unwrap_or_default();
```

Comment:
Suggesting simpler code:

```suggestion
    let mut properties = match orig.remove("properties") {
        Some(Value::Object(props)) => props,
        _ => Map::default(),
    };
```